src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -14,8 +14,15 @@
 #include <ucs/datastruct/string_buffer.h>
 #include <uct/ib/mlx5/rc/rc_mlx5.h>
 #include <uct/cuda/base/cuda_iface.h>
+#include <uct/cuda/cuda_copy/cuda_copy_md.h>
 
 #include <doca_log.h>
+#include <cuda_runtime.h>
```

Comment:
Please avoid using the CUDA Runtime API in UCT.
cudaMemset -> cuMemsetD8
cudaMemcpy -> cuMemcpyHtoD